legend-engine-language-pure-grammar/src/test/java/org/finos/legend/engine/language/pure/grammar/test/roundtrip/TestDomainGrammarRoundtrip.java

```diff
@@ -19,6 +19,28 @@
 
 public class TestDomainGrammarRoundtrip extends TestGrammarRoundtrip.TestGrammarRoundtripTestSuite
 {
+    @Test
```

Comment:
Not sure about this grammar.  Any reason why we this should not be modeled as a profile/tag value ? 

@pierredebelen might have a strong view on this one.

Generally, my preference is to keep the variations in syntax/grammar to a minimum.

Reply:
I bring over what we support in Pure, don't think we should diverge here. Also this is a native UML capability

Reply:
Yep it's part of the UML metamodel